packages/devextreme/testing/tests/DevExpress.ui.widgets/loadPanel.tests.js

```diff
@@ -59,37 +118,55 @@ QUnit.module('init', {
         assert.equal($content.find(MESSAGE_SELECTOR).text(), 'Test Loading Message');
     });
 
-    QUnit.test('role on wrapper', function(assert) {
-        const instance = $('#loadPanel').dxLoadPanel({ visible: true }).dxLoadPanel('instance');
+    QUnit.test('correct wrapper aria attributes load when showIndicator = false', function(assert) {
+        const label = messageLocalization.format('Loading');
+        const loadPanel = $('#loadPanel').dxLoadPanel({
+            visible: true,
+            showIndicator: false
+        }).dxLoadPanel('instance');
+
+        const $wrapper = loadPanel.$wrapper();
 
-        assert.strictEqual(instance.$wrapper().attr('role'), 'alert');
+        assert.strictEqual($wrapper.attr('role'), 'alert', 'role attribute has value alert ');
+        assert.strictEqual($wrapper.attr('aria-label'), label, 'aria-label attribute has value Loading...');
     });
 
-    QUnit.test('aria-label on wrapper', function(assert) {
-        const instance = $('#loadPanel').dxLoadPanel({ visible: true }).dxLoadPanel('instance');
+    QUnit.test('wrapper aria attributes must not load when showIndicator = true', function(assert) {
+        const $loadPanel = $('#loadPanel').dxLoadPanel({
+            visible: true,
+            showIndicator: true
+        });
+
+        assert.strictEqual($loadPanel.attr('role'), undefined, 'role is not set');
+        assert.strictEqual($loadPanel.attr('aria-label'), undefined, 'aria-label is not set');
+    });
```

Comment:
I see you test different behavior on init depending on `loadIndicator` option value => this functionality depends on `loadIndicator` option => we should also test the behavior after this option runtime change :)

Reply:
[7c89496](https://github.com/DevExpress/DevExtreme/pull/27080/commits/7c89496fe5ea9a3a695c7dcce3717057deeca65c)